### DISCUSSION TITLE
Personalize Wizarr

### DISCUSSION THREAD

**Discussion by djtomybcn** [wizarrrr/wizarr#443] [General]
Hi! I'm very surprised about this fantantic app. Congratulations!

I'm wondering if it's possible to personalize the frontend. I can't find information about this.

Thanks in advance!

**Comment by DJS91** [comment]
I assume you're talking about the Join page from the invitation url.
If you have some html/css knowhow you can easily edit the invitation pages yourself.

Once Wizarr is installed, you can update the page by editing the HTML files in /app/templates/ folder.
